go.mod

```diff
@@ -28,3 +28,12 @@ tool (
 	golang.org/x/tools/cmd/stringer
 	mvdan.cc/gofumpt
 )
+
+ignore (
+	node_modules
```

Comment:
When specified without `./`, this means "ignore `node_modules` at any level", which I thing is correct in general.